.github/workflows/try-pr-on-eclipse-che-hosted-by-red-hat.yaml

```diff
@@ -0,0 +1,15 @@
+on: pull_request_target
+
+jobs:
+  example_comment_pr:
+    runs-on: ubuntu-latest
+    name: Try the PR on Eclipse Che hosted by Red Hat
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@master
```

Comment:
Hi,
I would not use master as it may break at any time
Also probably we could just add a custom che or redhat action that do the work (so it could be easy to add that action on other repositories)

instead of copying/pasting the same message everywhere (let say we want to update title or badge, etc we would need to update on every repositories)

Reply:
+1 for not using master / main - we can switch to `https://github.com/thollander/actions-comment-pull-request/tree/1.0.1`

Could you clarify a bit the custom che / GH action approach? I though that the only way to do it is to create a yaml per repo, similar to what is currently done for the CI builds based on GH actions

Reply:
sure,

for example for happy path tests running with github actions, instead of having each repositories having lot of copy/paste instructions, there is a custom action that can be added

in your example we could replace

```yaml
- name: Comment PR
        uses: thollander/actions-comment-pull-request@1.0.1
        with:
          message: 'Try the PR on Eclipse Che hosted by Red Hat [![Contribute](https://img.shields.io/badge/Eclipse_Che-Hosted%20by%20Red%20Hat-525C86?logo=eclipse-che&labelColor=FDB940)](https://workspaces.openshift.com#https://github.com/${{ github.event.pull_request.head.repo.full_name }}/tree/${{ github.head_ref }})'
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
by

```yaml
- name: Hosted Che by Red Hat
        uses: <che-incubator or redhat-developer>/comment-pr@next
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

(the whole message to add, icons, etc is managed by the action)

so for example we see we should swap icon, text etc we just bump our action and it's updated everywhere
We could add a status check as well

example:
https://github.com/che-incubator/che-deploy-action
https://github.com/che-incubator/happy-path-tests-action


you may check https://github.com/benoitf/che-pr-check-gh-action

Reply:
This is nice, but don't you think it is overkill to create a separate action and maintain it on our own vs just having ~ 10 line file per-repo which is unlikely to be changed in the future?

Reply:
well it also add status check, you have properties to enable/disable, etc

Reply:
well, but this is a significant support burden for a simple badge generation that could be accomplished with an existing action. Not sure if it is really worth reinventing the action for this particular case e.g. - https://github.com/benoitf/che-pr-check-gh-action